tests/validation/tests/v3_api/test_bkp_restore.py

```diff
@@ -8,79 +8,142 @@
 
 namespace = {"p_client": None, "ns": None, "cluster": None, "project": None,
              "nodes": []}
+backup_info = {"backupname": None, "backup_id": None, "workload": None,
+               "backupfilename": None, "etcdbackupdata": None}
 
+@pytest.mark.run(order=1)
+def test_bkp_restore_local_create(create_project_client):
+    validate_backup_create("local")
 
-def test_bkp_restore_local(create_project_client):
-    validate_backup_create_restore_delete("local")
+
+@pytest.mark.run(order=2)
+def test_bkp_restore_local_restore():
+    validate_backup_restore("local")
+
```

Comment:
@izaac We should still have the ability to run each of the cases  relating to restore and delete individually if we choose to. Can you have these methods also include all the pre req steps needed like create for these test cases ?

Reply:
Ok I would have to re-design the splitting.  Will work on that.